person.rb

```diff
@@ -1,9 +1,46 @@
-class Person
```

Comment:
- [ ] Please specify the `Person` class with the `correct_name` method in it as per project requirements. 🙏

Reply:
@MahmoudBakr23 
Sorry but I don't see the error in my code, I think it fulfilled that 
https://github.com/JavierAybar/OOP-school-library/blob/b1dd42a54372b4558f8d563ff029b48e58d23ba7/person.rb#L53-L55

Requirement
Make sure that this class has a method correct_name implemented. It should simply return the name attribute.